File: sdks/typescript/src/namespaces/receipts.ts
import type { HttpClient } from "../http.js";

export class ReceiptsNamespace {
  constructor(private http: HttpClient) {}

  async get(id: string): Promise<unknown> {
    return this.http.request({
      method: "GET", path: `/v1/receipts/${encodeURIComponent(id)}`,
    });
  }
}

File: sdks/python/sealstack/namespaces/receipts.py
"""Receipts namespace (read-only)."""

from typing import Any

from .._http import HttpClient


class ReceiptsNamespace:
    def __init__(self, http: HttpClient) -> None:
        self._http = http

    async def get(self, id: str) -> Any:
        return await self._http.request("GET", f"/v1/receipts/{id}")
